repsetdata.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -euo pipefail
+
+OUTPUT_NAME="variables-by-replication-set-$(date +"%Y_%m_%d_%I_%M_%s").csv" 
+SELECT_STATEMENT=$(cat <<EOF
+        SELECT  
+          RTRIM(LTRIM(REPLACE(CAST(replication_sets AS varchar), '{}', 'Full' ), '{'),'}') AS "ReplicationSets", 
+          policy_id AS "Policy",
+          identifier(resource_id) as "Variable",
+        FROM resources 
+          WHERE 
+            policy_id NOT LIKE 'system%'
+          AND 
+            resource_id LIKE '%:variable:%'
+        ORDER BY replication_sets, resource_id ASC
+EOF
+)
+     
+parse_parameters() {
+  POSITIONAL_ARGS=()
+
+  while [[ $# -gt 0 ]]; do
+    case "${1:-}" in
+    --podman)
+      CONTAINERS_PLATFORM="podman"
+      shift ;;
+    --docker)
+      CONTAINERS_PLATFORM="docker"
+      shift ;;
+    -h | --help ) print_help
+      shift ;;
+    --) shift; 
+        break 
+        ;;
+    -*|--*)
+      echo "Unknown option $1"
+      exit 1
+      ;;
+    *)
+      POSITIONAL_ARGS+=("$1") # save positional arg
+      shift # past argument
+      ;;    
+    esac
+  done
+  if [ ${#POSITIONAL_ARGS[@]} -eq 1 ]; then
+      CONTAINER_NAME=${POSITIONAL_ARGS[0]}
```

Comment:
```suggestion
    CONTAINER_NAME=${POSITIONAL_ARGS[0]}
```

Should be unindented 1 level